src/main/java/Duke.java

```diff
@@ -1,10 +1,82 @@
+import java.util.Scanner;
+
 public class Duke {
+    static int MAX_SIZE = 100;
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        Scanner in = new Scanner(System.in);
+        Task[] tasks;
+        tasks = new Task[MAX_SIZE];
+        System.out.println("Hello! I'm Duke");
+        System.out.println("What can I do for you?");
+        int size = 0;
```

Comment:
This variable could perhaps have a different name - because it stores the number tasks. The name 'size' is a little less descriptive in the terms of the purpose of the variable.

Reply:
Updated. Thanks